xonsh/built_ins.py

```diff
@@ -578,12 +579,15 @@ def load_builtins(execer=None):
     builtins.__xonsh_all_jobs__ = {}
     builtins.__xonsh_active_job__ = None
     builtins.__xonsh_ensure_list_of_strs__ = ensure_list_of_strs
+    builtins.__history__ = History()
```

Comment:
This and `ordered_history` should probably be unloaded in the `unload_builtins` function in this same file, as well.